be/src/storage/lake/starlet_group_assigner.cpp

```diff
@@ -0,0 +1,39 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#include "storage/lake/starlet_group_assigner.h"
+
+#ifdef USE_STAROS
+
+#include <fmt/format.h>
+
+#include "service/staros_worker.h"
+
+namespace starrocks::lake {
+
+static const char* const kStarletPrefix = "staros_";
+
+StatusOr<std::string> StarletGroupAssigner::get_group(int64_t tablet_id) {
+    if (g_worker == nullptr) {
+        return Status::InternalError("init_staros_worker() must be called before get_shard_info()");
+    }
+    ASSIGN_OR_RETURN(auto shardinfo, g_worker->get_shard_info(tablet_id));
+    auto group_id = 1001; // TODO get group id from shardinfo
```

Comment:
where does group id comes from? group id is table id, shard info does not hold table id

Reply:
According to our earlier discussion, FE will save group id in ShardInfo as a custom property when creating shard.